Dockerfile

```diff
@@ -62,5 +62,5 @@ RUN virtualenv --no-site-packages -p python2 /opt/letsencrypt/venv && \
 # bash" and investigate, apply patches, etc.
 
 ENV PATH /opt/letsencrypt/venv/bin:$PATH
-# TODO: is --text really necessary?
```

Comment:
That comment was meant to make sure that `dialog` (ncurses) display works in Docker out of the box. I think I might have had some problems, back when I left the comment. Unfortunately, Docker is broken on my sid box, so I can't verify it. 

Did you actually manage to run the ncurses interface without any problems?